docs/python_docs/python/tutorials/packages/index.rst

```diff
@@ -46,9 +46,9 @@ Shared APIs
 
    .. card::
       :title: Symbol API
-      :link: symbol/index.html
+      :link: /api/python/docs/api/symbol/index.html
```

Comment:
isn't it better that this link is relative too?

Reply:
@larroy yes, that's a good point. Thank you for catching that. I've updated this reference to a relative link.